beets/ui/commands.py

```diff
@@ -1364,6 +1359,21 @@ def modify_items(lib, mods, dels, query, write, move, album, confirm):
             obj.try_sync(write, move)
 
 
+def print_modify_item(obj, mods, dels):
```

Comment:
I think `print_and_modify` implies that the function would really apply those changes.
What about `print_modify_changes`?